src/styles/product_card.scss

```diff
@@ -0,0 +1,118 @@
+
+$cardWidth: 200px;
+$gapX: 40px;
+$paddingX: 40px;
+$colums: 4;
+
+.maincon {
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 0;
+  padding: 46px 26px 46px 28px;
+  width: $colums * $cardWidth + ($colums - 1) * $gapX + 2 * $paddingX;
+
+  &__product-card {
+    margin-right: 48px;
+    margin-bottom: 48px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.cards {
+  width: 200px;
+  height: 406px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+
+  &__photo {
+    margin: 0 3px 36.5px;
+
+    &_product {
+      width: 100%;
+      height: 134px;
+    }
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    color: rgba(6, 11, 53, 1);
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    color: rgba(97, 96, 112, 1);
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    margin-bottom: 28px;
+    display: flex;
+    justify-content: space-between;
+
+    &_name {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: rgba(97, 96, 112, 1);
+    }
+
+    &_numb {
+      font-weight: 700;
+      font-size: 16px;
+      color: rgba(6, 11, 53, 1);
+      line-height: 18px;
+    }
```

Comment:
Use nesting only for elements of card, don't create nesting between separate elements